src/extra/python/isca/templates/mkmf.cheyenne.intel

```diff
@@ -0,0 +1,9 @@
+# template for the Intel fortran compiler version 8
```

Comment:
How different is this file from the default mkmf template?  Could it be achieved by putting things into the environment file instead?